parachains/runtimes/collectives/collectives-polkadot/src/fellowship/migrations.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Migrations.
+
+use frame_support::{pallet_prelude::*, traits::OnRuntimeUpgrade, weights::Weight};
+use log;
+
+/// Initial import of the Kusama Technical Fellowship.
+pub(crate) mod import_kusama_fellowship {
+	use super::*;
+	use frame_support::parameter_types;
+	use pallet_ranked_collective::{
+		Config, IdToIndex, IndexToId, MemberCount, MemberRecord, Members,
+		Pallet as RankedCollective, Rank,
+	};
+
+	const TARGET: &'static str = "runtime::migration::import_fellowship";
+
+	parameter_types! {
+		// The Fellowship addresses from Kusama state.
+		pub const FellowshipAddresses: [(Rank, [u8; 32]); 45] = [
+			(6, hex_literal::hex!("86b7409a11700afb027924cb40fa43889d98709ea35319d48fea85dd35004e64"),),
```

Comment:
script to export https://gist.github.com/muharem/8d21bfdbab6bf4ccf7fbdc9c3ad4d228